From fe344a4130f7e90705d224140a2e57997e051afb Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Sun, 3 Dec 2017 11:37:41 +0000
Subject: [PATCH 3/3] kernel/depmod: Use less efficient `-a` argument to catch
 all kernel modules

Looks like depmod itself is a bit silly and will miss some bits.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 src/handlers/kernel/depmod.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/handlers/kernel/depmod.c b/src/handlers/kernel/depmod.c
index a638607..cb38c11 100644
--- a/src/handlers/kernel/depmod.c
+++ b/src/handlers/kernel/depmod.c
@@ -38,7 +38,7 @@ static UscHandlerStatus usc_handler_depmod_exec(UscContext *ctx, const char *pat
         autofree(char) *kernel_nom = NULL;
         char *command[] = {
                 "/sbin/depmod",
-                "-A", /* only probe new fellers */
+                "-a", /* probe all fellers */
                 NULL, /* The path we're depmodding */
                 NULL, /* Terminator */
         };
-- 
2.15.1

